packages/ui/lib/components/AlbumPreview/index.tsx

```diff
@@ -9,24 +8,58 @@ import TrackRow from '../TrackRow';
 import artPlaceholder from '../../../resources/media/art_placeholder.png';
 import styles from './styles.scss';
 
-const AlbumPreview = ({
+type Artist = {
+  name: string;
+};
+
+type Track = {
+  album?: string;
+  artist?: Artist;
```

Comment:
?? check this once ..object or direct string?

Reply:
![image](https://user-images.githubusercontent.com/67220225/150933768-7e03684d-dcc1-413b-9ec1-dfa4b9888b66.png)
Originally its string only